src/App.jsx

```diff
@@ -56,6 +62,8 @@ function App() {
           </span>
         </div>
       </footer>
+        <ReactQueryDevtools initialIsOpen={true}/>
```

Comment:
the default element that is used is `footer` and it causes some styling issues. I suggest you set the container element to 'div'.